### PR TITLE
Adding viral assembly model with modified genome

### PR DESCRIPTION
**Pull request by brandonfung-93**

### CODE REVIEW

**Comment by mattsmart** [comment]
I can't immediately see the diff between the two files, but I think we only want the one in the viralassembly folder, right?

File: models/viralassembly/viralAssemblyModel.m [hunk 1]
```diff
@@ -1,170 +1,157 @@
...
-	%% Plot!
-	% This plots concentration of virions over time
-	figure(1);
-	plot(t, out(:,19),'LineWidth',2);
-	title('Concentration of Virions over Time')
-	xlabel('Time')
-	ylabel('Virions')
-	
+
+%% Initial Conditions
```
Comment (mattsmart) [inline models/viralassembly/viralAssemblyModel.m]: Can you tab all the lines within the function back out 4 or 8 spaces

File: models/viralassembly/viralAssemblyModel.m [hunk 2]
```diff
@@ -1,170 +1,157 @@
...
-	
-	% Complete virions
-	eq19 = k_anchor*P_3*V_i - k_v*V - delta_v*V; %ok
-	
-	% Altogether now!
-	dS = [eq1 eq2 eq3 eq4 eq5 eq6 eq7 eq8 eq9 eq10 eq11 eq12 eq13 eq14 eq15 eq16 eq17 eq18 eq19]';    
-	
-end
+
+%% Provide useful names for things
```
Comment (mattsmart) [inline models/viralassembly/viralAssemblyModel.m]: same here, tab em all out 4 or 8 spaces plz

File: models/viralassembly/viralAssemblyModel.m [hunk 3]
```diff
@@ -1,170 +1,157 @@
...
+% Complete pure virions
+eq17 = k_anchor*P_3*V_i - k_v*V - delta_v*V - v_exit*V;
+
+% Complete impure virions
+eq18 = k_anchor*P_3*V_mi - k_v*V_m - delta_v*V_m - v_exit*V_m; % Brandon to update
+
+% Altogether now!
+dS = [eq1 eq2 eq3 eq4 eq5 eq6 eq7 eq8 eq9 eq10 eq11 eq12 eq13 eq14 eq15 eq16 eq17 eq18]';    
+
+end
```
Comment (mattsmart) [inline models/viralassembly/viralAssemblyModel.m:157]: extra newline at end of file

**Comment by JeffGoldblum** [comment]
The reason why the tabs are gone is because MATLAB and/or GitHub is dumb. In MATLAB they don't distinguish between tab and four spaces. So when I write something in MATLAB sometimes I use tab and sometimes four spaces. GitHub does distinguish between tab and four spaces so my code looks like shit in the end because the four spaces will be look ok and the tabs will have way more white space.

On line 131 in the second term in that equation there's an ampersand instead of an asterisk.
I also have a few changes listed in HipChat, namely that the conversion rates of pure genomes to impure should be proportional to the concentration of pure genomes and that P5 can be translated from modified 35S RNA.

**Comment by mattsmart** [comment]
LGTM, merging

**Pull request merged**